lib/http-bridge/src/Cardano/Wallet/HttpBridge/Compatibility.hs

```diff
@@ -124,3 +127,18 @@ instance DecodeAddress (HttpBridge (network :: Network)) where
       where
         errBase58 = "Unable to decode Address: expected Base58 encoding."
         errDecoding _ = "Unable to decode Address: not a valid Byron address."
+
+-- | An initial first block to initialize a chain using the http-bridge. We do
+-- not use the `blockHash` and, do only use the `prevBlockHash` to catch up with
+-- unstable epoch and therefore, the very first `prevBlockHash` matters not.
+--
+-- It isn't impossible to retrieve the 'blockHash' by computing a blake2b 256 of
+-- the CBOR-serialized full block header, but this requires us to write the full
+-- CBOR decoders (and encoders) for the all BlockHeader which is, for the
+-- http-brdige implementation, a waste of time at the moment.
```

Comment:
In that case — looking at a chunk of blocks and getting the `prevBlockHash` of the child/next block might be an idea.